services/diversity/api_doc/functional.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @apiDefine s_richness Search > Species richness
+ * Information related to the species richness in a given area.
+ */
```

Comment:
Wrong group, this one is already defined and is missing correct description. (I see other group used from routes)

Reply:
Fixed